image/spec/specs-go/v1/image_deprecated.go

```diff
@@ -0,0 +1,28 @@
+package v1
+
+import dockerspec "github.com/moby/docker-image-spec/specs-go/v1"
```

Comment:
https://go.dev/wiki/CodeReviewComments#imports

Since the imported name gets rendered into the package godoc, we really should prefer not to rename the import in this package in particular.

Yes, that means the import should be named `v1`. You should have thought of that before naming the package v1 _in a brand new module that did not exist an hour ago_ 😠

Reply:
those package names were used to align with the same approach as the OCI specs.

I'll remove the alias for the import when I get back to my computer

Reply:
I would have preferred something like `specsv1` as ~everyone is going to alias it to anyway, but giving the `github.com/moby/docker-image-specs/specs-go/v1` package the name `v1` for consistency with `github.com/opencontainers/image-spec/specs-go/v1` is fair. https://github.com/opencontainers/image-spec/blob/365fa41472a19107b0b4cbc17e141a9b97f9b66e/specs-go/v1/annotations.go#L15

On the other hand, the package `github.com/moby/docker-image-spec/specs-go` being named `v1` is _inconsistent_ with the `github.com/opencontainers/image-spec/specs-go` package. https://github.com/opencontainers/image-spec/blob/365fa41472a19107b0b4cbc17e141a9b97f9b66e/specs-go/version.go#L15

Reply:
> name v1 for consistency with github.com/opencontainers/image-spec/specs-go/v1 is fair

I _hate_ the name, but yes, we chose that to be as close to "upstream" as possible.

> On the other hand, the package github.com/moby/docker-image-spec/specs-go being named v1 is inconsistent 

That one for sure was overlooked, and I missed it when migrating the code to the other repository. I don't know why I didn't spot it before, and when preparation was done before moving the code 🙈.

I guess that one's to "fix in post" perhaps in a next patch release of the module (assuming that changing that won't be a breaking change).